internal/buildjob/buildjob_test.go

```diff
@@ -0,0 +1,235 @@
+package buildjob
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/dominodatalab/forge/api/forge/v1alpha1"
+	"github.com/dominodatalab/forge/internal/config"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	testK8sClient "k8s.io/client-go/kubernetes/fake"
```

Comment:
I would have gone with fakeClient or just use fake since it doesn't overlap rather than a long name.